lib/formations.js

```diff
@@ -176,225 +199,263 @@ function formation_payload_check(payload, sizes) {
 }
 
 // public
-async function create(pg_pool, app_uuid, app_name, space_name, space_tags, org_name, type, size, quantity, command, port, healthcheck, remove_health_check, user = "System") {
-  assert.ok(app_uuid, 'The app uuid was not passed into create formations.')
-  assert.ok(type, 'The type was not passed in to formations create.')
-  assert.ok(space_name, 'The space_name was not passed in to formations create.')
-  assert.ok(space_tags || space_tags === "", 'The space_tags was not passed in to formations create.')
-  type = type.toLowerCase().trim()
-  if(app_name.indexOf('-') > -1) {
+async function create(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Async function 'create' has a complexity of 18. Maximum allowed is 8.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5244943)